src/components/forms/form-add-film.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Form, Input, Col } from 'reactstrap';
+import PropTypes from 'prop-types';
+
+import { MinutesInput } from './inputs/minutes-input';
+import { FormElement } from './form-element';
+
+const formElementAddFilm = [
+  {
+    id: 1,
+    as: Input,
+    type: 'text',
+    name: 'filmName',
+    label: 'Movie title',
+    rules: { required: true },
+    errorMessage: 'This field is required',
+  },
+  {
+    id: 2,
+    as: MinutesInput,
+    type: 'number',
+    name: 'duration',
+    label: 'Movie duration',
+    rules: { required: true },
+    errorMessage: 'This field is required',
+  },
+  {
+    id: 3,
+    as: Input,
+    type: 'text',
+    name: 'description',
+    label: 'Description',
+    rules: { required: true },
+    errorMessage: 'This field is required',
+  },
+  {
+    id: 4,
+    as: Input,
+    type: 'text',
+    name: 'genre',
+    label: 'Film genre',
+    rules: { required: true },
+    errorMessage: 'This field is required',
+  },
+  {
+    id: 5,
+    as: Input,
+    type: 'file',
+    name: 'file',
+    label: 'File',
+  },
+];
+
+export const FormAddFilm = ({
+  handleSubmit, errors, control,
+}) => {
+  const onSubmit = () => {};
+
+  return (
+    <Form onSubmit={handleSubmit(onSubmit)}>
```

Comment:
Why do you need this component at all? it's completely the same as AddCinema

Reply:
I made the same components cause when I made sign in and sign up forms Ilia told me to do two forms and two modals and I made this forms and modals similarly